### ISSUE TITLE
 corebird segfaults

### ISSUE THREAD

**Issue by innir** [baedert/corebird#750]
Hi,

another bug report from Debian, the full bug report is at: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=868136

```
Here is the output from coredumpctl info.

           PID: 32675 (corebird)
           UID: 1000 (edward)
           GID: 1000 (edward)
        Signal: 11 (SEGV)
     Timestamp: Wed 2017-07-12 11:49:58 BST (35min ago)
  Command Line: corebird
    Executable: /usr/bin/corebird
 Control Group: /user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service
          Unit: user@1000.service
     User Unit: gnome-terminal-server.service
         Slice: user-1000.slice
     Owner UID: 1000 (edward)
       Boot ID: eef0b2d167b448018be0036c898d6e08
    Machine ID: 8afb2a74abdf4af0899f95f7fffc781c
      Hostname: x1
       Storage: /var/lib/systemd/coredump/core.corebird.1000.eef0b2d167b448018be0036c898d6e08.32675.1499856598000000.lz4
       Message: Process 32675 (corebird) of user 1000 dumped core.
                
                Stack trace of thread 32675:
                #0  0x00007f47c67c3b80 g_utf8_offset_to_pointer (libglib-2.0.so.0)
                #1  0x00007f47c67c3c10 g_utf8_substring (libglib-2.0.so.0)
                #2  0x000055acfb4e7197 n/a (corebird)
                #3  0x000055acfb4e7496 n/a (corebird)
                #4  0x000055acfb4e8236 n/a (corebird)
                #5  0x000055acfb478f0e n/a (corebird)
                #6  0x000055acfb47d3dc n/a (corebird)
                #7  0x000055acfb485154 n/a (corebird)
                #8  0x00007f47c6a6fd27 g_cclosure_marshal_VOID__BOXEDv (libgobject-2.0.so.0)
                #9  0x00007f47c6a6d1a4 _g_closure_invoke_va (libgobject-2.0.so.0)
                #10 0x00007f47c6a878cd g_signal_emit_valist (libgobject-2.0.so.0)
                #11 0x00007f47c6a87fbf g_signal_emit (libgobject-2.0.so.0)
                #12 0x00007f47c90ee422 n/a (libsoup-2.4.so.1)
                #13 0x00007f47c90ee84d n/a (libsoup-2.4.so.1)
                #14 0x00007f47c90ef485 n/a (libsoup-2.4.so.1)
                #15 0x00007f47c90ef6db n/a (libsoup-2.4.so.1)
                #16 0x00007f47c67938ca g_main_dispatch (libglib-2.0.so.0)
                #17 0x00007f47c6793c80 g_main_context_iterate (libglib-2.0.so.0)
                #18 0x00007f47c6793d2c g_main_context_iteration (libglib-2.0.so.0)
                #19 0x00007f47c6d5029d g_application_run (libgio-2.0.so.0)
                #20 0x000055acfb46f173 n/a (corebird)
                #21 0x00007f47c63ca2b1 __libc_start_main (libc.so.6)
                #22 0x000055acfb46f00a n/a (corebird)
                
                Stack trace of thread 32677:
                #0  0x00007f47c648963d poll (libc.so.6)
                #1  0x00007f47c6793c16 g_main_context_poll (libglib-2.0.so.0)
                #2  0x00007f47c6793d2c g_main_context_iteration (libglib-2.0.so.0)
                #3  0x00007f47bad5446d n/a (libdconfsettings.so)
                #4  0x00007f47c67bb4f5 g_thread_proxy (libglib-2.0.so.0)
                #5  0x00007f47c59d4494 start_thread (libpthread.so.0)
                #6  0x00007f47c6492a8f __clone (libc.so.6)
                
                Stack trace of thread 32678:
                #0  0x00007f47c648963d poll (libc.so.6)
                #1  0x00007f47c6793c16 g_main_context_poll (libglib-2.0.so.0)
                #2  0x00007f47c6793d2c g_main_context_iteration (libglib-2.0.so.0)
                #3  0x00007f47c6793d71 glib_worker_main (libglib-2.0.so.0)
                #4  0x00007f47c67bb4f5 g_thread_proxy (libglib-2.0.so.0)
                #5  0x00007f47c59d4494 start_thread (libpthread.so.0)
                #6  0x00007f47c6492a8f __clone (libc.so.6)
                
                Stack trace of thread 32679:
                #0  0x00007f47c648963d poll (libc.so.6)
                #1  0x00007f47c6793c16 g_main_context_poll (libglib-2.0.so.0)
                #2  0x00007f47c6793fa2 g_main_loop_run (libglib-2.0.so.0)
                #3  0x00007f47c6d7c286 gdbus_shared_thread_func (libgio-2.0.so.0)
                #4  0x00007f47c67bb4f5 g_thread_proxy (libglib-2.0.so.0)
                #5  0x00007f47c59d4494 start_thread (libpthread.so.0)
                #6  0x00007f47c6492a8f __clone (libc.so.6)

This is from GDB.

Core was generated by `corebird'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  g_utf8_offset_to_pointer (
    str=0x55ad000ce0f0 "nitedutilities @AliDriverUK @RoryStewartUK replied to GM last night with this Guy, some work IS already happening through #WoodlandTrust, 65k trees planted: https://t.co/igufpf0rRV 🌳", offset=4294238837) at ././glib/gutf8.c:351
351	././glib/gutf8.c: No such file or directory.
[Current thread is 1 (Thread 0x7f47c96c8480 (LWP 32675))]
(gdb) bt
#0  0x00007f47c67c3b80 in g_utf8_offset_to_pointer (str=0x55ad000ce0f0 "nitedutilities @AliDriverUK @RoryStewartUK replied to GM last night with this Guy, some work IS already happening through #WoodlandTrust, 65k trees planted: https://t.co/igufpf0rRV 🌳", offset=4294238837) at ././glib/gutf8.c:351
#1  0x00007f47c67c3c10 in g_utf8_substring (str=<optimized out>, start_pos=0, end_pos=4294967294) at ././glib/gutf8.c:285
#2  0x000055acfb4e7197 in  ()
#3  0x000055acfb4e7496 in  ()
#4  0x000055acfb4e8236 in  ()
#5  0x000055acfb478f0e in  ()
#6  0x000055acfb47d3dc in  ()
#7  0x000055acfb485154 in  ()
#8  0x00007f47c6a6fd27 in g_cclosure_marshal_VOID__BOXEDv (closure=0x55acff391df0, return_value=<optimized out>, instance=0x55acfd8df1c0, args=<optimized out>, marshal_data=<optimized out>, n_params=<optimized out>, param_types=0x55acff45f0e0) at ././gobject/gmarshal.c:1950
#9  0x00007f47c6a6d1a4 in _g_closure_invoke_va (closure=closure@entry=0x55acff391df0, return_value=return_value@entry=0x0, instance=instance@entry=0x55acfd8df1c0, args=args@entry=0x7ffc9cb56dd0, n_params=<optimized out>, param_types=0x55acff45f0e0) at ././gobject/gclosure.c:867
#10 0x00007f47c6a878cd in g_signal_emit_valist (instance=0x55acfd8df1c0, signal_id=<optimized out>, detail=0, var_args=var_args@entry=0x7ffc9cb56dd0)
    at ././gobject/gsignal.c:3300
#11 0x00007f47c6a87fbf in g_signal_emit (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>) at ././gobject/gsignal.c:3447
#12 0x00007f47c90ee422 in  () at /usr/lib/x86_64-linux-gnu/libsoup-2.4.so.1
#13 0x00007f47c90ee84d in  () at /usr/lib/x86_64-linux-gnu/libsoup-2.4.so.1
#14 0x00007f47c90ef485 in  () at /usr/lib/x86_64-linux-gnu/libsoup-2.4.so.1
#15 0x00007f47c90ef6db in  () at /usr/lib/x86_64-linux-gnu/libsoup-2.4.so.1
#16 0x00007f47c67938ca in g_main_dispatch (context=0x55acfd72f500) at ././glib/gmain.c:3234
#17 0x00007f47c67938ca in g_main_context_dispatch (context=context@entry=0x55acfd72f500) at ././glib/gmain.c:3899
#18 0x00007f47c6793c80 in g_main_context_iterate (context=context@entry=0x55acfd72f500, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>)
    at ././glib/gmain.c:3972
#19 0x00007f47c6793d2c in g_main_context_iteration (context=context@entry=0x55acfd72f500, may_block=may_block@entry=1) at ././glib/gmain.c:4033
#20 0x00007f47c6d5029d in g_application_run (application=0x55acfd8dd120 [Corebird], argc=1, argv=0x7ffc9cb592a8) at ././gio/gapplication.c:2381
#21 0x000055acfb46f173 in  ()
#22 0x00007f47c63ca2b1 in __libc_start_main (main=
    0x55acfb46efd0, argc=1, argv=0x7ffc9cb592a8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffc9cb59298) at ../csu/libc-start.c:291
#23 0x000055acfb46f00a in  ()
```

**Comment by Vistaus** [comment]
I see that it happens with a DM. Most likely the same issue as https://github.com/baedert/corebird/issues/747 which should be fixed in 1.5.1 so please update to 1.5.1 or tell Debian to update you to 1.5.1

**Comment by innir** [comment]
Thanks for the quick reply, I updated the Debian package.
